### PR TITLE
add tests for 5 ops based on model zoo

### PR DESCRIPTION
**Pull request by bszmelcz**
Add tests for:
* binary_convolution
* group_convolution
* ctc_greedy_decoder
* region_yolo
* reorg_yolo

### CODE REVIEW

File: inference-engine/tests/functional/inference_engine/ngraph_reader/binary_convolution_tests.cpp [hunk 1]
```diff
@@ -0,0 +1,448 @@
...
+			<output>
+				<port id="0" precision="FP32">
+					<dim>1</dim>
+					<dim>64</dim>
+					<dim>56</dim>
+					<dim>56</dim>
+				</port>
+			</output>
+		</layer>
+		<layer id="1" name="612" type="MaxPool" version="opset1">
```
Comment (ilyachur) [inline inference-engine/tests/functional/inference_engine/ngraph_reader/binary_convolution_tests.cpp:23]: Can we remove unnecessary operations from tests?

File: inference-engine/tests/functional/inference_engine/ngraph_reader/binary_convolution_tests.cpp [hunk 2]
```diff
@@ -0,0 +1,448 @@
+// Copyright (C) 2018-2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <string>
+#include "ngraph_reader_tests.hpp"
+
+TEST_F(NGraphReaderTests, ReadBinaryConvolutionNetwork) {
```
Comment (ilyachur) [inline inference-engine/tests/functional/inference_engine/ngraph_reader/binary_convolution_tests.cpp:8]: From my point test on the binary convolution should test only binary convolution.

File: inference-engine/tests/functional/inference_engine/ngraph_reader/region_yolo_tests.cpp [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+			<output>
+				<port id="1" precision="FP32">
+					<dim>1</dim>
+					<dim>1</dim>
+					<dim>1</dim>
+					<dim>1</dim>
+				</port>
+			</output>
+		</layer>
+		<layer id="2" name="yolo_out_postprocess" type="Multiply" version="opset1" precision="FP32">
```
Comment (ilyachur) [inline inference-engine/tests/functional/inference_engine/ngraph_reader/region_yolo_tests.cpp:34]: I think we don't need this Multiply in the test for region yolo.

File: inference-engine/tests/functional/inference_engine/ngraph_reader/reorg_yolo_tests.cpp [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+			<output>
+				<port id="1" precision="FP32">
+					<dim>1</dim>
+					<dim>104</dim>
+					<dim>32</dim>
+					<dim>13</dim>
+				</port>
+			</output>
+		</layer>
+		<layer id="2" name="yolov2/yolov2_feature/lower_features/downsample/YoloRegion" type="RegionYolo" version="opset1">
```
Comment (ilyachur) [inline inference-engine/tests/functional/inference_engine/ngraph_reader/reorg_yolo_tests.cpp:42]: Can we remove Region Yolo?

File: inference-engine/tests/functional/inference_engine/ngraph_reader/ctc_greedy_decoder_tests.cpp [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+
+#include <string>
+#include "ngraph_reader_tests.hpp"
+
+TEST_F(NGraphReaderTests, ReadCTCGreedyDecoderNetwork) {
+    std::string model = R"V0G0N(
+<net name="ctcgreedydecoder" version="10">
+	<layers>
+		<layer id="0" name="CTCGreedyDecoder/placeholder_port_0" type="Parameter" version="opset1">
+			<data shape="20,8,128" element_type="f32"/>
```
Comment (sdurawa) [inline inference-engine/tests/functional/inference_engine/ngraph_reader/ctc_greedy_decoder_tests.cpp:13]: I think we shall add another test case when attribute merge_repeated=false is defined.

File: inference-engine/tests/functional/inference_engine/ngraph_reader/ctc_greedy_decoder_tests.cpp [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+			<data offset="0" size="640" shape="20,8" element_type="f32"/>
+			<output>
+				<port id="0" precision="FP32">
+					<dim>20</dim>
+					<dim>8</dim>
+				</port>
+			</output>
+		</layer>
+		<layer id="2" name="CTCGreedyDecoder" type="CTCGreedyDecoder" version="opset1">
+			<data ctc_merge_repeated="1"/>
```
Comment (sdurawa) [inline inference-engine/tests/functional/inference_engine/ngraph_reader/ctc_greedy_decoder_tests.cpp:32]: I think we shall add another test case when attribute merge_repeated=false is defined.

File: inference-engine/tests/functional/inference_engine/ngraph_reader/region_yolo_tests.cpp [hunk 7]
```diff
@@ -0,0 +1,150 @@
+// Copyright (C) 2018-2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <string>
+#include "ngraph_reader_tests.hpp"
+
+TEST_F(NGraphReaderTests, ReadRegionYoloNetwork) {
```
Comment (sdurawa) [inline inference-engine/tests/functional/inference_engine/ngraph_reader/region_yolo_tests.cpp:8]: New test case shall be added with following attributes: do_softmax=false and proper mask value.

File: inference-engine/tests/functional/inference_engine/ngraph_reader/binary_convolution_tests.cpp (hunk 1 above)
Comment (bszmelcz) [reply inference-engine/tests/functional/inference_engine/ngraph_reader/binary_convolution_tests.cpp:23]: Will investigate it today

File: inference-engine/tests/functional/inference_engine/ngraph_reader/region_yolo_tests.cpp (hunk 3 above)
Comment (bszmelcz) [reply inference-engine/tests/functional/inference_engine/ngraph_reader/region_yolo_tests.cpp:34]: yup, done

File: inference-engine/tests/functional/inference_engine/ngraph_reader/reorg_yolo_tests.cpp (hunk 4 above)
Comment (bszmelcz) [reply inference-engine/tests/functional/inference_engine/ngraph_reader/reorg_yolo_tests.cpp:42]: done

File: inference-engine/tests/functional/inference_engine/ngraph_reader/binary_convolution_tests.cpp (hunk 1 above)
Comment (bszmelcz) [reply inference-engine/tests/functional/inference_engine/ngraph_reader/binary_convolution_tests.cpp:23]: done

**Review by ilyachur — APPROVED** [review]

**Pull request merged**